packages/components/src/internal/components/files/FilePreviewGrid.tsx

```diff
@@ -28,6 +28,7 @@ export class FilePreviewGrid extends React.Component<Props, any> {
                 ) : (
                     <>
                         <strong>{header}</strong>
+                        <Alert bsStyle="warning">{warningMsg}</Alert>
```

Comment:
Looking at this in the UI, it might be worth adding a little top padding so that this is spaced out from the header.

Reply:
Can do.  Was just following the styling in the spec.